cypress/e2e/tests/navbar.cy.js

```diff
@@ -208,14 +208,10 @@ context('Global Docs and API Ref', () => {
     cy.location('pathname').should('include', '/apis/global/')
   })
 
-  it('cy.reload() - reload the page', () => {
-    cy.reload(true)
-  })
-
   it('visits /apis/global/ directly', () => {
 
-    cy.visit('//apis/global/', {
-      timeout: 50000, // increase total time for the visit to resolve
+    cy.visit('/apis/global/', {
+      timeout: 120000, // increase total time for the visit to resolve
```

Comment:
That's long timeout. What's causing this delay, any idea?

Reply:
As mentioned our Slack, please create a placeholder ticket to discuss it further. As you alluded to, any such longer wait times indicates to a poor UX and should be addressed.